avogadro/qtplugins/gamessinput/gamessinput.cpp

```diff
@@ -24,9 +24,9 @@
 
 #include <molequeue/client/jobobject.h>
 
-#include <QtCore/QDebug>
-
-#include <QtWidgets/QMessageBox>
+#include <QAction>
+#include <QDebug>
```

Comment:
Does this require that we remove the Qt module, or just the addition of the QAction include?

Reply:
No, the fix is just adding the missing QAction header. But with Qt5 the recommendation is to not use the Qt module prefix anymore, so I drop wherever I see them.

Reply:
Fair enough, we use it elsewhere so I would prefer not to drop  it, I can append to your commit and merge that though.

Reply:
I can force-push just the QAction change in a second if you want me to

Reply:
That would be awesome if you don't mind